tensorflow/lite/micro/python/interpreter/tests/interpreter_test.py

```diff
@@ -143,7 +143,10 @@ def testCompareWithTFLite(self):
     tflm_interpreter = tflm_runtime.Interpreter.from_bytes(model_data)
 
     # TFLite interpreter
-    tflite_interpreter = tf.lite.Interpreter(model_content=model_data)
+    tflite_interpreter = tf.lite.Interpreter(
```

Comment:
Does removing tolerance in line 175 now work?

Reply:
Yes. Removed. Again the real fix will be #1463